go.mod

```diff
@@ -11,13 +11,13 @@ require (
 	github.com/container-orchestrated-devices/container-device-interface v0.4.0
 	github.com/containernetworking/cni v1.1.0
 	github.com/containernetworking/plugins v1.1.1
-	github.com/containers/buildah v1.25.2-0.20220423102655-8f2bb8876f3f
-	github.com/containers/common v0.47.5-0.20220425182415-4081e6be9356
+	github.com/containers/buildah v1.26.1
+	github.com/containers/common v0.48.0
 	github.com/containers/conmon v2.0.20+incompatible
-	github.com/containers/image/v5 v5.21.1-0.20220425080628-be085685e524
-	github.com/containers/ocicrypt v1.1.3
+	github.com/containers/image/v5 v5.21.1
+	github.com/containers/ocicrypt v1.1.4-0.20220428134531-566b808bdf6f
```

Comment:
It bothers me to vendor a non-release. I'll check where this comes from.

Reply:
@rhatdan can you vendor v1.4.0 (https://github.com/containers/ocicrypt/releases/tag/v1.1.4)?